stable/nginx-ingress/templates/controller-deployment.yaml

```diff
@@ -22,6 +25,9 @@ spec:
         component: "{{ .Values.controller.name }}"
         release: {{ .Release.Name }}
     spec:
+{{- if .Values.controller.rbac.enabled }}
+      serviceAccountName: {{ template "fullname" . }}
+{{- end }}
```

Comment:
@jsulinski hmmm strange the Traefik chart doesn't have any complex logic here, the switch is pretty much identical.